### PR TITLE
Update plot_kernel_smoothing.py

### PR DESCRIPTION
**Pull request by ElenaPetrunina**
* Fix runtime warning (division by zero)
* Change bandwidth values in CV parameter search
* Reorder sections

### CODE REVIEW

File: examples/plot_kernel_smoothing.py [hunk 1]
```diff
@@ -3,70 +3,142 @@
...
+# smooth them as a preprocessing step.
 #
 # As an example, we will smooth the first 300 curves only. In the following
 # plot, the first five curves are shown.
+
 dataset = skfda.datasets.fetch_phoneme()
 fd = dataset['data'][:300]
 
-fd[:5].plot()
+out = fd[:5].plot()
```
[the remark this answers is not in the document]
Comment (ElenaPetrunina) [reply examples/plot_kernel_smoothing.py]: Fixed, thanks.

File: examples/plot_kernel_smoothing.py [hunk 2]
```diff
@@ -3,70 +3,141 @@
...
-import matplotlib.pylab as plt
+import matplotlib.pyplot as plt
 import numpy as np
 
 import skfda
-import skfda.misc.hat_matrix as hm
 import skfda.preprocessing.smoothing.validation as val
 from skfda.preprocessing.smoothing import KernelSmoother
+from skfda.misc.kernels import uniform
+from skfda.misc.hat_matrix import (
```
[the remark this answers is not in the document]
Comment (ElenaPetrunina) [reply examples/plot_kernel_smoothing.py:20]: I reviewed the imports (and run isort in local) and they looks ordered to me. I am missing something?

[the remark this answers is not in the document]
Comment (vnmabus) [reply examples/plot_kernel_smoothing.py:20]: I will merge it and take a look at that later.

**Review by vnmabus — APPROVED** [review]

File: examples/plot_kernel_smoothing.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ElenaPetrunina) [reply examples/plot_kernel_smoothing.py:20]: Perfect, thanks

**Pull request merged**